### ISSUE TITLE
Remove Natwar and Narora from Indian states

### ISSUE THREAD

**Issue by apmuthu** [hiiamrohit/Countries-States-Cities-database#88]
````
(18, 'Kenmore', 101),
..
(27, 'Narora', 101),
(28, 'Natwar', 101),
..
(30, 'Paschim Medinipur', 101),
..
(40, 'Vaishali', 101),
````
The above are not states in India.

**Comment by UtkarshGpta** [comment]
Fixed in PR #94. Can be closed.

**Comment by apmuthu** [comment]
Ok. Closed
